pallets/subspace/src/subnet.rs

```diff
@@ -160,6 +140,26 @@ impl<T: Config> Pallet<T> {
         Ok(())
     }
 
+    pub fn subnet_params(netuid: u16) -> SubnetParams<T> {
```

Comment:
For sake of consistency, it'd be nice to have `check_subnet_params` be below this function, no?